src/app/backend/dashboard.go

```diff
@@ -86,7 +89,15 @@ func main() {
 	// TODO(maciaszczykm): Move to /appConfig.json as it was discussed in #640.
 	http.Handle("/api/appConfig.json", handler.AppHandler(handler.ConfigHandler))
 	http.Handle("/metrics", prometheus.Handler())
-	log.Print(http.ListenAndServe(fmt.Sprintf("%s:%d", *argBindAddress, *argPort), nil))
+
+	// Listen for http and https
+	addr := fmt.Sprintf("%s:%d", *argBindAddress, *argPort)
+	go log.Fatal(http.ListenAndServe(addr, nil))
```

Comment:
In case of secure setup, you probably want to limit the bind andress to localhost for the insecure port